WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCard.swift

```diff
@@ -1,7 +1,87 @@
 import Yosemite
 import SwiftUI
 
-struct CollapsibleProductRowCard: View {
+/// Displays a single collapsible product row or grouped parent and child product rows
+struct CollapsibleProductCard: View {
```

Comment:
I like this implementation with a container of the existing `CollapsibleProductRowCard` 👍